script.js

```diff
@@ -0,0 +1,462 @@
+            rusShiftButtonPic.insertAdjacentText('afterbegin', rusShiftButtonsPic[i]);
+            i++;
+            let paragraph = document.createElement('p');
+            paragraph.insertAdjacentText('beforeend', 'Клавиатура создана в операционной системе Windows. Для переключения языка комбинация: левыe shift + alt ');
+            buttonElement.appendChild(buttonImg);
+            buttonElement.appendChild(rusButtonImg);
+            buttonElement.appendChild(shiftButton);
+            buttonElement.appendChild(rusShiftButton);
+            buttonElement.appendChild(shiftButtonPic);
+            buttonElement.appendChild(rusShiftButtonPic);
+
+
+            switch(button) {
+                case "backspace":
+                    buttonElement.classList.add('backspace');
+                    buttonElement.addEventListener('click', () => {
+                        this.properties.value = this.properties.value.substring(0, this.properties.value.length - 1);
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "\\":
+                    buttonElement.classList.add('backslash');
+                    buttonElement.addEventListener('click', () => {
+                        if (this.properties.ru == 'false') {
+                            this.properties.capsLock == false ? this.properties.value += buttonElement.textContent.substring(0,1).toLowerCase() : this.properties.value += buttonElement.textContent.substring(0,1);
+                        }
+                        else if (this.properties.ru == 'true') {
+                            this.properties.capsLock == false ? this.properties.value += buttonElement.textContent.substring(1,2).toLowerCase() : this.properties.value += buttonElement.textContent.substring(1,2);
+                        }        
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "tab":
+                    buttonElement.classList.add('tab');   
+                    buttonElement.addEventListener('click', () => {
+                        this.properties.value += '\t';
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "CapsLock":
+                    buttonElement.classList.add('caps');
+                    buttonElement.addEventListener('mousedown', () => {
+                        this.properties.capsLock == true ? this.properties.capsLock = false : this.properties.capsLock = true;
+                        this.properties.capsLock == true ? document.getElementsByClassName(`caps`)[0].style.backgroundColor = 'coral': document.getElementsByClassName(`caps`)[0].style.backgroundColor = '';
```

Comment:
document.getElementsByClassName(`caps`)[0] - you should request any element directly several times. You can just put it into a variable and work with it